.gitlab/deploy_packages/nix.yml

```diff
@@ -187,3 +187,10 @@ deploy_staging_iot_agent:
     - $S3_CP_CMD $S3_ARTIFACTS_URI/iot/agent ./agent
     - export PACKAGE_VERSION=$(inv agent.version --url-safe --major-version 7)
     - $S3_CP_CMD ./agent $S3_DSD6_URI/linux/iot/agent-$PACKAGE_VERSION --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
+
+# Remote Updater
+deploy_remote_updater_arm64:
```

Comment:
could we use a matrix for deploy jobs?

Reply:
We might want to look into it indeed, that sounds like a good way of simplifying those jobs